### PR TITLE
Csharp 3ps

### PR DESCRIPTION
**Pull request by ankit-privado**

### CODE REVIEW

File: rules/sinks/storages/microsoft/caching/csharp.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+sinks:
```
Comment (HiteshMah-Jan) [inline rules/sinks/storages/microsoft/caching/csharp.yaml:1]: why rule for local cache ?
cache makes sense only if the code runs on end-user devices, like JS or android apps

File: rules/sinks/third_parties/sdk/componentspace/csharp.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.SDK.ComponentSpace
+    name: Component Space
```
Comment (HiteshMah-Jan) [inline rules/sinks/third_parties/sdk/componentspace/csharp.yaml:8]: single word - ComponentSpace

File: rules/sinks/third_parties/sdk/identitymodel/csharp.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (HiteshMah-Jan) [inline rules/sinks/third_parties/sdk/identitymodel/csharp.yaml:1]: not a third-party

File: rules/sinks/storages/microsoft/caching/csharp.yaml (hunk 1 above)
Comment (ankit-privado) [reply rules/sinks/storages/microsoft/caching/csharp.yaml:1]: yes right will remove them.

**Review by HiteshMah-Jan — APPROVED** [review]

**Pull request merged**